src/renderer/lib/beat_util.ts

```
@@ -16,6 +16,42 @@
+    }
+  }
+  if (beat.htmlPrompt) {
+    return "Html Prompt";
+  }
+  return "Image Prompt";
+};
+
+export const getMediaIcon = (type: string) => {
+  switch (type) {
+    case "video":
+      return Video;
```

ESLint

[error] 'Video' is not defined.

(no-undef)